src/Ledger.Net/Responses/Abstract/GetPublicKeyResponseBase.cs

```diff
@@ -23,6 +24,7 @@ protected GetPublicKeyResponseBase(byte[] data) : base(data)
                 PublicKeyData = memoryStream.ReadAllBytes(publicKeyLength);
                 var addressLength = memoryStream.ReadByte();
                 Address = GetAddressFromStream(memoryStream, addressLength);
+                ChainCodeData = memoryStream.ReadAllBytes(32);
```

Comment:
The EthereumAppGetPublicKeyRequest has an option for including the chain code data or not, which you could think is potentially problematic to include it into the GetPublicKeyResponseBase. However, even if the chain code is not included, this line will not result in an error. The stream will just read in 32 bytes of value 0.